docs/UserGuide.md

```diff
@@ -441,7 +460,7 @@ To edit the details of a specific book, you can enter the `edit` command with th
 
 **Notes about the edit command:**<br>
 
-* Edits the book at the specified `INDEX`. `INDEX` refers to the index number shown in the displayed book list. The index **must be a positive integer** 1, 2, 3, … 
+* Edits the book at the specified `INDEX`. `INDEX` refers to the index number shown in the displayed book list. The index **must be a positive integer** 1, 2, 3, … and cannot exceed the largest index number in the displayed book list.
```

Comment:
Good consideration in the UG to ensure that the supplied index doesn't exceed the largest index in the displayed book list 👍